src/modules/portfolio/components/market-portfolio-card/market-portfolio-card-footer.jsx

```diff
@@ -119,13 +125,18 @@ const MarketPortfolioCardFooter = p => {
 MarketPortfolioCardFooter.propTypes = {
   linkType: PropTypes.string.isRequired,
   localButtonText: PropTypes.string.isRequired,
-  buttonAction: PropTypes.func,
-  outstandingReturns: PropTypes.string,
-  finalizationTime: PropTypes.number,
+  buttonAction: PropTypes.func.isRequired,
+  outstandingReturns: PropTypes.string.isRequired,
+  finalizationTime: PropTypes.number.isRequired,
```

Comment:
`finalizationTime` is undefined in MarketPortfolioCardFooter when used from MarketPortfolioCard.